spec/models/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user1) { build(:user) }
+  let(:user2) { build(:user) }
+  let(:user3) { build(:user) }
+  let(:friendship1) { create(:friendship, requestor: user1, acceptor: user2) }
+  let(:friendship2) { create(:friendship, requestor: user1, acceptor: user3) }
+  let(:friendship3) { create(:friendship, requestor: user2, acceptor: user3) }
+
+  describe 'validations' do
+    context 'complete required info' do
+      it 'is valid' do
+        expect(user1).to be_valid
+      end
+    end
+
+    # context 'username is missing' do
+    #   it 'is not valid' do
+    #     user1.update(username: '')
+    #     expect(user1.errors[:username]).to include("can't be blank")
+    #   end
+    # end
+  end
+
+  describe 'friendships' do
```

Comment:
Why are we writing specs for `friendships` here in `user_spec.rb` (lines 28 to 56)?